src/SmartComponents/SystemTable/SystemTable.js

```diff
@@ -153,6 +153,7 @@ const SystemTable = ({ bulkSelectIds, selectedIds, selectIds }) => {
         isStickyHeader: true,
         onSelect: items.length ? selectIds : null,
       }}
+      showCentosVersions={true}
```

Comment:
It would probably also be enough to have just `showCentosVersions`, without the `={true}`.